Buildings/Resources/Scripts/Dymola/Controls/OBC/Utilities/PIDWithAutotuning/AutoTuner/Amigo/Validation/PI.mos

```diff
@@ -0,0 +1,4 @@
+simulateModel("Buildings.Controls.OBC.Utilities.PIDWithAutotuning.AutoTuner.AMIGO.Validation.PI", method="Cvode", tolerance=1e-06, resultFile="PI");
```

Comment:
Make sure the .mos files path containing "AMIGO", not "Amigo"

Reply:
fixed it